app/packs/src/apps/mydb/elements/list/ElementsTableSampleEntries.js

```diff
@@ -107,6 +107,22 @@ const showDecoupledIcon = (sample) => (sample.decoupled ? (
   </OverlayTrigger>
 ) : null);
 
+const showInventoryLabelIcon = (sample) => (sample.inventory_sample && sample.xref.inventory_label ? (
+  <div>
+    <OverlayTrigger
+      placement="top"
+      overlay={<Tooltip id="sample_inventory_label">Inventory Label</Tooltip>}
+    >
+      <Badge
+        className="sample-entries-inventory-label-icon"
```

Comment:
Please use bootstrap utility classes "p-1 mt-1 rounded text-dark" in this case. Use css classes only when there is no bootstrap alternative for the required property.

Reply:
thank you. I have refactored it. this is how it looks now
![image](https://github.com/user-attachments/assets/78a4b2b0-44ba-4aec-915e-6622d1278424)